publisher-sdk/src/main/java/com/criteo/publisher/CriteoAdListener.java

```diff
@@ -16,12 +16,14 @@
 
 package com.criteo.publisher;
 
+import androidx.annotation.Keep;
 import androidx.annotation.NonNull;
 import androidx.annotation.UiThread;
 
 /**
  * All callbacks are invoked on the UI thread, so it is safe to execute any UI operations in the implementation.
  */
+@Keep
```

Comment:
I'm not sure if this interface should be kept. Publishers are using the banner/interstitial listeners. They should not care about this interface that is only there to factorize few callbacks.
But I'm not sure if it is fully safe to remove the `@Keep` because of the default. Do you mind testing this without the `@Keep` ? I'm even wondering if we cannot just make this interface package private.

Reply:
Removed the `Keep` annotation.  I think making it private or not should be done in a separate PR though.